mindsdb/integrations/handlers/s3_handler/README.md

```diff
@@ -100,4 +100,179 @@ This table will return all objects regardless of the file format, however, only
         * Incorrect: SELECT * FROM integration.travel/travel_data.csv
         * Incorrect: SELECT * FROM integration.'travel/travel_data.csv'
         * Correct: SELECT * FROM integration.\`travel/travel_data.csv\`
-</Warning>
\ No newline at end of file
+</Warning>
```

Comment:
Let's update this doc for now you can add an example with the custom config. If we keep it like this it looks like we duplicate the same page